charts/liquibase-migration/values.yaml

```diff
@@ -1,53 +1,106 @@
-# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
-#                                                                                     #
-#   DEMONSTRATION: This file is NOT used because this is a library chart.             #
-#   The source of truth is in the templates themselves, this file is documentation.   #
-#                                                                                     #
-#   All unset values are required.                                                    #
-#                                                                                     #
-# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+# -- Specific migrations to run; each config merged over the defaults to form the full configuration
+# @default -- None by default, one entry here required per migration
+enumerated:
```

Comment:
nit: Call this `migrations`, `migrationJobs`, or `migrationDatabases` instead? `enumerated` feels a tad unclear 🤔